lib/utils.js

```diff
@@ -486,10 +487,22 @@ async function verifyApplicationPlatform (app, isSimulator) {
                   `Make sure the correct deployment target has been selected for its compilation in Xcode.`);
 }
 
+/**
+ * Return true if the platformName is tvOS
+ * @param {string} platformName The name of platorm
+ * @returns {boolean} Return true if the platformName is tvOS
+ */
+function isTvOS (platformName) {
+  const memoIsTvOS = _.memoize(function memoizedIsTvOS () {
```

Comment:
there is no point to use memoize here since it won't help to save any computation time, but would rather add more complexity.